docs/source/architecture_decisions.md

```diff
@@ -4,23 +4,23 @@ This is a collection of records for architecturally significant decisions.
 
 ### URL Patterns
 
-We follow a very basic yet strong convention for URL so that our rest APIs are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
+We follow a very basic ,yet strong convention for URL so that our rest API's are properly namespaced. First of all, we rely heavily on HTTP verbs to perform **CRUD** actions.
 
 For example, to perform **CRUD** operation on _Challenge Host Model_, following will be the URL patterns.
 
-* `GET /hosts/challenge_host_team` - Retrieves a list of challenge host teams
+* `GET /hosts/challenge_host_team` - Retrieves a list of challenge host teams,
```

Comment:
Can you please remove the comma as I think earlier version(without comma) was looking good . :-)